sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeFormatter.scala

```diff
@@ -100,8 +118,11 @@ private class CodeFormatter {
       indentString
     }
     code.append(f"/* ${currentLine}%03d */ ")
-    code.append(thisLineIndent)
-    code.append(line)
+    if (line.trim().length > 0) {
```

Comment:
Sure! This `if` makes blank lines print **only line numbers**, e.g.,
**Before**
```
"/* 001 */ <space><space><space><space> ..."
```
**After**
```
"/* 001 */ "
```
As you see, the redundant spaces for indentation go away.